doc/source/ray-air/examples/torch_incremental_learning.ipynb

```diff
@@ -526,10 +525,9 @@
     "      [transforms.ToTensor(), transforms.Normalize((0.1307,), (0.3081,))]\n",
     "    )\n",
     "\n",
-    "    df.loc[:, \"image\"] = TensorArray([\n",
-    "        torchvision_transforms(np.asarray(image)).numpy()\n",
-    "            for image in df[\"image\"]\n",
-    "    ])\n",
+    "    df.loc[:, \"image\"] = [\n",
+    "        torchvision_transforms(image).numpy() for image in df[\"image\"]\n",
+    "    ]\n",
```

Comment:
@jiaodong Note that this is tensor extension type agnostic, but we still need to convert the Torch tensors back to NumPy ndarrays before storing them in the batch.

Reply:
ack, having UDF purely tensor type agnostic and working with numpy + df is already big dex ux improvement :) I think modern DL frameworks all have good zero-copy numpy iterop so it should be much less concerning.

Reply:
im curious what would happen to existing user code / example / tests if they still wrapped output / input with `TensorArray`, or they're just working with Ragged tensor but also didn't explicitly disable auto casting ?